### PR TITLE
improve(release): create release

### PR DESCRIPTION
**Pull request by mrice32**
**Motivation**

To release recent smart contract functional and address changes to match the chain.

**Summary**

This is just the result of running `yarn create-release`.

**Issue(s)**

N/A

### CODE REVIEW

File: packages/core/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,21 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.1.0](https://github.com/UMAprotocol/protocol/compare/@uma/core@1.0.0...@uma/core@1.1.0) (2020-10-05)
+
```
Comment (chrismaree) [inline packages/core/CHANGELOG.md:7]: these are auto-generated, right? pretty cool how it pools from the issue name!

**Review by chrismaree — APPROVED** [review]
LGTM! really like the auto-generated messages 👏

**Review by daywiss — COMMENTED** [review]
This is fully automatic? If so thats really cool and looks good

**Review by daywiss — APPROVED** [review]

**Comment by mrice32** [comment]
> This is fully automatic? If so thats really cool and looks good

Yep, this is 100% automated. I did no editing :).

**Pull request merged**